editoast/src/views/timetable.rs

```diff
@@ -449,21 +458,49 @@ impl Conflict {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
-struct PacedTrainOccurrenceId {
-    paced_train_id: i64,
-    index: u64,
+#[serde(untagged)]
+enum ConflictPacedTrainId {
+    Occurence {
+        paced_train_id: i64,
```

Comment:
If `paced_train_id` is part of every variant (and it probably should), maybe the design of that type is incorrect. It should probably be a `struct`, that contains a `paced_train_id`, and a second field being the `enum`.